CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## Source AMI release notes
+- [Amazon Linux 2023 release notes](https://docs.aws.amazon.com/linux/al2023/release-notes/relnotes.html)
+- [Amazon Linux 2 release notes](https://docs.aws.amazon.com/AL2/latest/relnotes/relnotes-al2.html)
+- [Amazon Linux AMI 2018.03 Release Notes](https://aws.amazon.com/amazon-linux-ami/2018.03-release-notes/) 
+
+## 20230428
+- ecs version: [1.71.0-1](https://github.com/aws/amazon-ecs-agent/releases/tag/v1.71.0)
+- al2 docker version: 20.10.22
```

Comment:
should we add AL1, AL2023 docker and containerd versions?

Reply:
and that AL1 ecs agent version is 1.51.0?

maybe AL1, AL2 and AL2023 can be divided into 3 different sections for clearer picture..

Reply:
I didn't add them to the changelog because there is no Docker and containerd version updates on AL1 and AL2023 in this release. (Have not figured out a good way to present non-change info on this changelog)